Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM ubuntu:focal
+
+ARG USER_ID
+ARG GROUP_ID
+
+RUN apt-get update && \
+	apt-get install -y wget \
+		build-essential \
+		rsync \
+		pax
+
+RUN wget https://github.com/debbuild/debbuild/archive/20.04.0.tar.gz && \
+	tar -zxvf 20.04.0.tar.gz && \
+	cd debbuild-20.04.0 && \
+	./configure && \
+	make && \
+	make install && \
+	cd / && \
+	rm -r debbuild-20.04.0 20.04.0.tar.gz
```

Comment:
Why we cannot use debbuild from repository?

Reply:
Which repository? I can't fund `debbuild` in XUbuntu 20.04 repos.

Reply:
You can use precompiled https://github.com/debbuild/debbuild/releases/download/21.01.0/debbuild_21.01.0-0ubuntu20.04_all.deb instead of compile.

Reply:
This change is unrelated to RA-2585. Let's do it in scope of another PR.

Reply:
Why? You added build script in this ticket.